e2e/test/scenarios/embedding-sdk/static-dashboard-cors.cy.spec.js

```diff
@@ -91,9 +91,7 @@ describeSDK("scenarios > embedding-sdk > static-dashboard", () => {
 
     cy.get("#metabase-sdk-root").within(() => {
       cy.findByText("Error").should("be.visible");
-      cy.findByText(
-        "Could not authenticate: invalid JWT URI or JWT provider did not return a valid JWT token",
-      ).should("be.visible");
+      cy.findByText(/Failed to fetch the user/).should("be.visible");
```

Comment:
hard to say, I'd rather have the full error message anyway. This isn't like we're changing the error every PR or anything. Not having tests failed could be bad sometimes.

Since we also have tests that assert the full sentence. It might not be of a concern.